# HG changeset patch
# User Vytautas Janƒçauskas <unaudio@gmail.com>
# Date 1379440715 -10800
#      Tue Sep 17 20:58:35 2013 +0300
# Node ID b3399e03eb210383204e45bf17c2efea35d1b247
# Parent  d24bf3fb4c09a9be93a4abe6d4eb7463bd2807d8
doc: short descriptions in audiodevinfo.txi and audiofile.txi for the functions

diff --git a/doc/interpreter/audiodevinfo.txi b/doc/interpreter/audiodevinfo.txi
--- a/doc/interpreter/audiodevinfo.txi
+++ b/doc/interpreter/audiodevinfo.txi
@@ -1,4 +1,6 @@
 @node Audio Device Information
 @section Audio Device Information
 
+This function is used to get information about audio devices that are available on your system. This information can then be used when creating audioplayer and audiorecorder objects.
+
 @DOCSTRING(audiodevinfo)
\ No newline at end of file
diff --git a/doc/interpreter/audiofile.txi b/doc/interpreter/audiofile.txi
--- a/doc/interpreter/audiofile.txi
+++ b/doc/interpreter/audiofile.txi
@@ -1,8 +1,11 @@
 @node Audio File Utilities
 @section Audio File Utilities
 
 The following functions provide functionality for reading and writing audio files in various formats including .wav, .flag, .ogg and many others. Also for retrieving information about audio files, such as number of channels, sampling rate, metadata and other.
 
 @DOCSTRING(audioinfo)
+
+The following two functions read and write audio files.
+
 @DOCSTRING(audioread)
 @DOCSTRING(audiowrite)
\ No newline at end of file
